python_purify/core.py

```diff
@@ -97,7 +97,7 @@ def _call_method(self, method, **kwargs):
             extra=extra
         )
         if self._verbose:
-            print url
+            print(url)
```

Comment:
https://python-future.org/compatible_idioms.html#print

Reply:
please add `from __future__ import print_function` so we don't loose anything with py2